example/index.js

```diff
@@ -0,0 +1,154 @@
+import createGraph from 'ngraph.graph';
+import ForceGraph from './force-graph';
+import { Broadcast } from '..';
+import { addPeer as _addPeer, removePeer as _removePeer, findPeer, nodesToArray } from './utils';
+
+const MAX_PEERS = 2;
+const TOPIC = Buffer.from('batman');
+
+const graph = createGraph();
+const peersTitle = document.getElementById('peers-title');
+const connectionsTitle = document.getElementById('connections-title');
+const packetsSendedTitle = document.getElementById('packets-sended-title');
+const packetsReadedTitle = document.getElementById('packets-readed-title');
+const view = ForceGraph()(document.getElementById('graph'));
+
+const changePeerColor = (peer, color) => {
+  const node = graph.getNode(peer.id.toString('hex'));
+  if (!node) return;
+  node.data.color = color;
+  graph.addNode(node.id, node.data);
+};
+
+const delay = ms => new Promise(resolve => setTimeout(resolve, ms));
+
+let packetsSended = 0;
+let packetsReaded = 0;
+
+const addPeer = () => {
+  const peer = _addPeer(graph, TOPIC, {
+    bootstrap: ['https://signal.wireline.ninja']
```

Comment:
Move to config file? Seems odd to have this in code?
Or TODO to move and reference the signal project and have the example set-up require a local server?